src/rgw/rgw_process.cc

```diff
@@ -106,13 +106,13 @@ int rgw_process_authenticated(RGWHandler_REST * const handler,
     return ret;
   }
 
-  ldpp_dout(op, 2) << "pre-executing" << dendl;
+  ldpp_dout(op, 2) << "pre-executing, latency=" << s->time_elapsed() << dendl;
   op->pre_exec();
 
-  ldpp_dout(op, 2) << "executing" << dendl;
+  ldpp_dout(op, 2) << "executing, latency=" << s->time_elapsed() << dendl;
   op->execute();
 
-  ldpp_dout(op, 2) << "completing" << dendl;
+  ldpp_dout(op, 2) << "completing, latency=" << s->time_elapsed() << dendl;
```

Comment:
time_elapsed() is already included in this log output because of `ldpp_dout()` and `req_state::gen_prefix()`:

> 2018-11-07 10:09:50.918 7f7f157fa700  2 req 1 **0.002s** s3:list_buckets verifying op params
2018-11-07 10:09:50.918 7f7f157fa700  2 req 1 **0.002s** s3:list_buckets pre-executing
2018-11-07 10:09:50.918 7f7f157fa700  2 req 1 **0.002s** s3:list_buckets executing
2018-11-07 10:09:50.919 7f7f157fa700  2 req 1 **0.003s** s3:list_buckets completing
2018-11-07 10:09:50.919 7f7f157fa700  2 req 1 **0.003s** s3:list_buckets op status=0
2018-11-07 10:09:50.919 7f7f157fa700  2 req 1 **0.003s** http status=200